spiketoolkit/validation/validation_list.py

```diff
@@ -0,0 +1,63 @@
+from .quality_metric_classes.metric_data import MetricData
```

Comment:
I would remove the classes from the init, it makes things more confusing for the user IMO

Reply:
Yes, that is fair.

Reply:
Fixed